src/test/qgstest.h

```diff
@@ -292,13 +296,64 @@ class TEST_EXPORT QgsTest : public QObject
       return result;
     }
 
+    bool checkLongStr( const char *file, const char *function, int line, const QString &name, const QString &referenceName, const QByteArray &actualStr )
+    {
+      QString subPath = "control_images/" + mControlPathPrefix + "/expected_" + name + "/" + "expected_" + referenceName;
+      QString expectedPath = testDataPath( subPath );
+      QFile expectedFile( expectedPath );
+      if ( ! expectedFile.open( QFile::ReadOnly  | QIODevice::Text ) )
+        return false;
+      QByteArray expectedStr = expectedFile.readAll();
+
+      if ( actualStr.size() != expectedStr.size() )
+        qWarning() << "Array have not the same length (actual vs expected): " << actualStr.size() << "vs" << expectedStr.size();
```

Comment:
```suggestion
        qWarning() << "Array have not the same length (actual vs expected): " << actualStr.size() << " vs " << expectedStr.size();
```

Reply:
`<<` operator already handle the spaces. By the way `(actual vs expected): " <<` has a extra space :)